lib/chef/provider/package/rubygems.rb

```diff
@@ -555,7 +555,7 @@ def install_via_gem_command(name, version)
           if new_resource.source.is_a?(String) && new_resource.source =~ /\.gem$/i
             name = new_resource.source
           else
-            src << "--clear-sources" if new_resource.clear_sources
+            src << "--clear-sources" if new_resource.clear_sources?
```

Comment:
was this meant to be changed?

Reply:
Yes, that's changing from checking the property value to calling this method https://github.com/chef/chef/pull/9480/files#diff-151fa83ee17ddfc007da9fe30edc0f91R59-R65

Reply:
it should be just clear_sources? then.

Reply:
Also if this is a method used in the provider you'll want to toss it in there